File: April 15, 2026.js
function sortAndSwap(arr) {

  let newArr = [...arr].sort((a, b) => a - b);

  for (let i = 0; i < newArr.length; i++) {
    if ((i % 3 == 0) && (i+1 <= newArr.length) && (i != 0)) {
      let aux = newArr[i];
      newArr[i] = newArr[i-1];
      newArr[i-1] = aux;
    }
  }

  return newArr;
}

File: April 15, 2026.py
def sort_and_swap(arr):

    new_arr = sorted(arr)
    for i in range(len(new_arr)):
        if i % 3 == 0 and i + 1 <= len(new_arr) and i != 0:
            aux = new_arr[i]
            new_arr[i] = new_arr[i-1]
            new_arr[i-1] = aux

    return new_arr